packages/lesswrong/lib/truncateUtils.ts

```diff
@@ -0,0 +1,105 @@
+import type { RefObject } from "react";
```

Comment:
Everything in this file has been refactored out of `TruncatedAuthorsList` and made slightly more generic so it can be reused to create `TruncatedTagsList` using the same method.